perfkitbenchmarker/providers/docker/docker_container_instance.py

```diff
@@ -278,8 +280,8 @@ def GetResourceMetadata(self):
       dict mapping string property key to value.
     """
     result = super(DockerContainer, self).GetResourceMetadata()
-    logging.warn("GET RESOURCE METADATA")
-    print(result)
+    logging.warn('GET RESOURCE METADATA')
+
```

Comment:
@dPhanekham was this leftover debug code?  I've removed it.  You could add it as to the above logging statement if it's useful.  But it does seem to be more info than warning.

Reply:
Yes, sorry, that was debug code that I didn't catch when cleaning up the file